### PR TITLE
testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Fix maintenance
- Upgrade to 6.3.1

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/config [hunk 1]
```diff
@@ -62,12 +62,12 @@ get__free_footprint() {
 set__maintenance_mode() {
     if [ "$maintenance_mode" -eq "1" ]; then
         # If maintenance_mode was set to 1, enable maintenance mode
-        (cd "$final_path" && ynh_exec_as "$app" \
+        (cd "$install_dir" && ynh_exec_as "$app" \
             echo "Site under maintenance." > .maintenance)
         ynh_print_info "Maintenance mode disabled"
```
Comment (tituspijean) [inline scripts/config]:
```suggestion
        ynh_print_info "Maintenance mode enabled"
```

File: scripts/config [hunk 2]
```diff
@@ -62,12 +62,12 @@ get__free_footprint() {
...
     elif [ "$maintenance_mode" -eq "0" ]; then
         # If maintenance_mode was set to 0, disable maintenance mode
-        ynh_secure_remove --file=$final_path/.maintenance
+        ynh_secure_remove --file=$install_dir/.maintenance
         ynh_print_info "Maintenance mode enabled"
```
Comment (tituspijean) [inline scripts/config]:
```suggestion
        ynh_print_info "Maintenance mode disabled"
```

File: scripts/config [hunk 3]
```diff
@@ -62,12 +62,12 @@ get__free_footprint() {
 set__maintenance_mode() {
     if [ "$maintenance_mode" -eq "1" ]; then
         # If maintenance_mode was set to 1, enable maintenance mode
-        (cd "$final_path" && ynh_exec_as "$app" \
+        (cd "$install_dir" && ynh_exec_as "$app" \
             echo "Site under maintenance." > .maintenance)
```
Comment (tituspijean) [inline scripts/config]:
```suggestion
        ynh_exec_as "$app" echo "Site under maintenance." > $install_dir/.maintenance
```

**Review by tituspijean — DISMISSED** [review]
Additionally, somehow the `__PHPVERSION__` in the FPM config file gets replaced by `7.4` instead of `8.2` while running the config panel, breaking the app. I have found no explanation for it.

File: scripts/config (hunk 3 above)
Comment (tituspijean) [reply scripts/config]: Oh, I did not check files permissions with my altered command.

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Review by alexAubin — APPROVED** [review]

**Pull request merged**